Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM node:14-buster-slim
+
+WORKDIR /app
+
+COPY package.json package-lock.json ./
+
+RUN npm install --production
+
+COPY layouts layouts
+COPY lib lib
+COPY views views
+COPY sg/db.json sg/
```

Comment:
You should minimize `COPY` commands as each creates a new layer. I think the traditional way to do this is `COPY . wherever` coupled with a `.dockerignore` file. That's at least what I did for html-build.